assets/js/partners/contact.js

```diff
@@ -43,6 +44,24 @@ async function submitPartnerContact(event) {
   const lastName = fetchTextInput('last_name');
   const jobTitle = fetchTextInput('job_title');
   const email = fetchTextInput('email').toLowerCase();
+
+  const blockedDomains = [
+    'gmail.com',
+    'hotmail.com',
+    'icloud.com',
+    'live.com',
+    'outlook.com',
+    'sbcglobal.net',
+    'yahoo.com',
+  ];
```

Comment:
If the idea is to limit submissions to a "government email address", should we target those as _allowed_, rather than blocking specific domains?

For example:

```ts
const isValidEmail = /\.(gov|mil)$/.test(email);
```

Reply:
@aduth not all state/local governments use `.gov` emails, so we don't want to be too restrictive. This is really just meant to prevent users from submitting support requests to partnerships and to limit spam in general.